frontend/admin/src/js/components/user/UserProfilePrintButton.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button } from "@common/components";
+import { commonMessages } from "@common/messages";
+import React, { useRef } from "react";
+import { useIntl } from "react-intl";
+import { useReactToPrint } from "react-to-print";
+import { Scalars, useGetUserProfileQuery } from "../../api/generated";
+import UserProfileDocument from "./UserProfileDocument";
+
+export interface UserProfilePrintButtonProps {
+  userId: Scalars["ID"];
+}
+
+export const UserProfilePrintButton: React.FunctionComponent<{
+  userId: Scalars["ID"];
+}> = ({ userId, children }) => {
+  const intl = useIntl();
+
+  // would be nice to only fire this if the button was clicked but you would need an imperative version of the query that you could await on
```

Comment:
If anyone knows how to get synchronous queries or a promise then I could hold back the query. But since the query is the same one as in the tab, URQL caches the results and only queries once anyway.